proknow/Patients/Patients.py

```diff
@@ -28,7 +29,10 @@ def _query(self, workspace, query):
         res, data = self._requestor.get('/workspaces/' + workspace.id + '/patients', params=query)
         if res.headers['proknow-has-more'] == 'true': # pragma: no cover (difficult to test w/o lg num of patients)
             next_query = dict(query)
-            next_query["next"] = res.headers['proknow-next']
+            if "page_newest" not in next_query:
+                next_query['page_newest'] = datetime.utcnow().isoformat()
+
+            next_query["page_number"] = res.headers['proknow-next-page']
```

Comment:
Currently, `page_newest` is being set when you query the second page. Why wouldn't you want to set this for the first page? Wouldn't it be possible to miss items that would otherwise have been on the second, third, fourth, ... pages?

Reply:
I would recommend changing `page_newest` to `page_epoch`. If `page_epoch` is unset, the API should get the current timestamp and use that for the query. The datetime should be passed to the client in a `ProKnow-Epoch` header. The client should use this as the `page_epoch` in subsequent queries.